source/whats-new.txt

```diff
@@ -28,8 +28,28 @@ Learn what's new in:
 What's New in 4.6
 -----------------
 
-See `v4.6.0 Release Highlights <https://github.com/mongodb/node-mongodb-native/releases/tag/v4.6.0>`__
-on GitHub.
+New features of the 4.6 Node.js driver release include:
+
+TypeScript: ``ChangeStreamDocument``
```

Comment:
Issue: not sure the technical details are properly communicated.

Suggestion: Given our recent discussion about how we should avoid trying to teach something in the What's New section and go into too much detail, remove this section on ChangeStreamDocument changes and link to the GitHub tag notes. For the link description, explain what [this ticket ](https://jira.mongodb.org/browse/NODE-4059) fixes.